tests/regression/57-floats/12-subtraction_assignmen.c

```diff
@@ -0,0 +1,11 @@
+//PARAM: --enable ana.float.interval
+
```

Comment:
```suggestion
#include assert.h

```
It seems like a `t` is missing at the end of the filename.